source/index.html

```diff
@@ -9,6 +9,7 @@
     </head>
     <body>
         <div id="mount"></div>
+        <script src="./player.js" type="text/javascript"></script>
```

Comment:
Why do we need to import `player.js` here when you've already imported into our `index.js`? Our build server is cool, and will bundle everything together, so we don't need this line here.